src/home_robot/home_robot/datasets/scannet/scannet_constants.py

```diff
@@ -10411,6 +10412,564 @@
     1175,
 )
 
+
+# Equivalent to:
+# labels_pd = pd.read_csv('meta_data/scannetv2-labels.combined.tsv', sep='\t', header=0)
+# labels_pd.loc[labels_pd.raw_category == 'stick', ['category']] = 'object'
+# labels_pd.loc[labels_pd.category == 'wardrobe ', ['category']] = 'wardrobe'
+# CLASS_ID_TO_NAME = dict(zip(labels_pd['id'], labels_pd['category']))
+CLASS_ID_TO_NAME = {
```

Comment:
Should we point to a specific scannet version to make sure this stays in sync? Not that i expect it to change or anything

Reply:
Thing that changes seems to be the number of classes, and whether those classes come from the scannet annotators or else MP40 or NYU40 classes. That mapping is in this [csv](https://github.com/daveredrum/ScanRefer/blob/9d7483053e8d29acfd4db4eb1bc28f1564f5dddb/data/scannet/meta_data/scannetv2-labels.combined.tsv), and we might want to restrict our attention to some subset of the full scannet categories (that's what this constants file is supposed to do). 

Eventually we probably want to pin the evaluation config for the dataset (e.g. frame skip, image size, etc) but I think we can do that in configs